src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((firstLeg * secondLeg) * 0.5);
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + this.getClass().getSimpleName().toLowerCase() + ", "
+                + "area: " + this.calculateArea() + " sq. units, "
+                + "firstLeg: " + this.firstLeg + ", "
+                + "secondLeg: " + this.secondLeg + ", "
+                + "color: " + getColor().toString()
+                + '}';
```

Comment:
The `toString` method should ensure consistent punctuation. Consider adding a space before the closing curly brace to maintain consistency in the output format.